newIDE/app/src/GameDashboard/Feedbacks/FeedbackCard.js

```diff
@@ -5,8 +5,6 @@ import { t, Trans } from '@lingui/macro';
 import { I18n } from '@lingui/react';
 import { type I18n as I18nType } from '@lingui/core';
 
-import Card from '@material-ui/core/Card';
```

Comment:
Yay! Every time we replace a material-ui component by a higher level component from our UI folder, we improve the app consistency.